portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,44 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> comments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String json = new Gson().toJson(comments);
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Jack!</h1>");
+    response.getWriter().println(json);
+  }
+
+@Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
```

Comment:
nit/optional: Param names are usually snake_case or camelCase. 

text_input or textInput

Reply:
Better yet - user_comment or userComment might be better here.

Reply:
got it

Reply:
@augerm fixed comments- can you unblock my merge?